src/app/confirm-dialog/confirm-dialog.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
import { ComponentFixture, TestBed } from "@angular/core/testing";
```